### DISCUSSION TITLE
Once Merge Protection is turned on, how do you turn it off?

### DISCUSSION THREAD

**Discussion by jodastephen** [Mergifyio/mergify#5118] [Q&A] [answered]
I pressed the big blue button on one of my repos: 
![image](https://github.com/user-attachments/assets/a466b8e7-66b7-434a-9667-cf03fae1f0ae)

Now every PR has an extra check:
![image](https://github.com/user-attachments/assets/0c347072-0191-49ca-8053-4251d1fa3a38)

I'd like to "deactivate" merge Protection, but there doesn't seem to be a way to do this. Can anyone help/advise?

(More generally, Merge Protection turned out to be unpleasant as it adds a comment to the PR, which I don't want)

**Comment by jd** [comment, marked as answer]
👋 @jodastephen 
Hey there! You can deactivate it from Integrations -> GitHub

Is only the comment the issue? That's probably something we could make optional.

**Comment by jodastephen** [comment]
Thanks for letting me know how to disable. Would be good to add that to the docs. 

Adding config to disable the comment, and/or make it templated would be good and would probably make me switch. I'm using the post_check instead, which is really weird to setup (you need a condition that is always true!), but much more flexible ATM.

One of my goals is to check the title for a JIRA reference and update the PR description by adding a hyperlink to the issue. But right now there is no way to update the description.

**Reply by jd** [reply]
Great, I'll make sure we have a switch to disable commenting at some point.

How would you update the description? What would be the hyperlink you add? A fixed one?

**Reply by jodastephen** [reply]
The logic we currently use via a GHA bot is to take the JIRA reference from the start of the title (by regex) and then insert it into an appropriate hyperlink. The PR body is updated with the hyperlink,  checking to see if it is already there to avoid duplicates:
![image](https://github.com/user-attachments/assets/137f3410-02ba-43c2-8f60-ac84caa1a11f)

I think the ability to update the body of a PR with a trailing message would be a good enhancement. (A comment causes email/notification spam from GH whereas updating the body does not)
